scripts/SMD_chip_package_rlc-etc/size_definitions/size_resistor_chip_smaller_0603.yaml

```diff
@@ -0,0 +1,12 @@
+R_0402:
+    code_imperial: "0402"
+    code_metric: "1005"
+    body_length:
+      minimum: 1.0
+      maximum: 1.1
+    body_width:
+      minimum: 0.48
+      maximum: 0.6
+    terminator_spacing_min: 0.4
+    terminator_spacing_max: 0.7
```

Comment:
Why not have `terminal_length` here?
And why use the old style?

Reply:
As mentioned above, probably we should consolidate but I didn't touch things that were outside of the scope of correcting C and L body size if I could help it. Since I wrote more about this above, I'll mark this comment as resolved and we can discuss in the other comment about this topic.